ArcGISRuntimeSDKQt_CppSamples/Maps/SetMaxExtent/README.metadata.json

```
@@ -0,0 +1,28 @@
+{
+    "category": "Maps",
+    "description": "Limit the view of a map to a particular area.",
+    "ignore": false,
+    "images": [
+        "screenshot.png"
+    ],
+    "keywords": [
+        "envelope",
+        "extent",
+        "limit panning",
+        "map",
+        "mapview",
+        "max extent",
+        "zoom"
```

```suggestion
        "extent",
        "limit panning",
        "max extent",
        "zoom",
        "Envelope",
        "Map",
        "MapView"
```
class names should follow the tags and be capitalized